docs/03-tutorials/00-eventing/evnt-04-change-max-in-flight-in-sub.md

```diff
@@ -2,24 +2,18 @@
 title: Changing Events Max-In-Flight in Subscriptions
 ---
 
-In this tutorial, we will show how to set idle "in-flight messages" limit in Kyma Subscriptions.
-The "in-flight messages" config defines the number of events which will be forwarded in parallel to the sink by Eventing Services without waiting for a response. 
+In this tutorial, you learn how to set idle "in-flight messages" limit in Kyma Subscriptions.
+The "in-flight messages" config defines the number of events that Eventing forwards in parallel to the sink, without waiting for a response. 
 
 ## Prerequisites
 
-1. Provision a [Kyma Cluster](../../02-get-started/01-quick-install.md).
-2. (Optional) Deploy [Kyma Dashboard](../../01-overview/main-areas/ui/ui-01-gui.md) on the Kyma cluster using the following command. Alternatively, you can also use `kubectl` CLI.
-   ```bash
-   kyma dashboard
-   ```
-3. (Optional) Install [CloudEvents Conformance Tool](https://github.com/cloudevents/conformance) for publishing events. Alternatively, you can also use `curl` to publish events.
-   ```bash
-   go install github.com/cloudevents/conformance/cmd/cloudevents@latest
-   ```
+Follow the [prerequisites steps](../../02-get-started/04-trigger-workload-with-event.md#prerequisites) mentioned in the getting-started guide.
+
+## Create a Function
 
-## Create a Workload
+First, create a sample Function that prints out the received event to console.
+To simulate prolonged event processing, in this example, the Function waits for 5 seconds before returning the response.
 
```

Comment:
The difference between the following instructions and the ones in `../../02-get-started/04-trigger-workload-with-event.md#create-a-function` is only the sample code, right?

If yes, then you could replace this whole section with something like:

```suggestion
2. Create a Function by following the [instructions](../../02-get-started/04-trigger-workload-with-event.md#create-a-function) in the Getting Started guide.

> **NOTE:** For this example, instead of the default code sample, replace the Function source with the following code:
```

Reply:
updated